### DISCUSSION TITLE
/posts/2023/08/17%E6%B5%85%E8%B0%88%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8E%E5%8E%9F%E7%90%86

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#26] [Announcements]
# /posts/2023/08/17%E6%B5%85%E8%B0%88%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8E%E5%8E%9F%E7%90%86

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2023/08/17%E6%B5%85%E8%B0%88%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8E%E5%8E%9F%E7%90%86

**Comment by a90120411** [comment]
感谢博主分享，读完对搜索引擎的工作流程有了一个全面的了解。
向您请教一下，如果想深入了解其中的技术，是否有推荐的书籍资料？

**Reply by Justin3go** [reply]
朋友你好，实话我也没有特别推荐的书籍，因为搜索引擎我并没有看过网上的书籍，相关知识是学生时代时学校老师传授的，学校的内部资料我也不太好分享出来。这本书[《这就是搜索引擎》](https://weread.qq.com/web/reader/a28325105a618ea287e580ake4d32d5015e4da3b7fbb1fa)我大致看了看目录，搜索引擎的相关知识基本都涉及到了，也许你可以试试阅读这本书

**Reply by a90120411** [reply]
谢谢推荐，祝您生活愉快！

**Reply by Justin3go** [reply]
hhhh，客气了🤝
